src/main/java/io/devcon5/vertx/services/AbstractService.java

```diff
@@ -124,6 +124,7 @@ public T withConfig(final JsonObject config) {
      */
     protected void start(final Future<Void> startFuture) {
 
+        System.out.println("you shall not use system.out.println");
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/devcon5-evaluation/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ch.devcon5.vertx%3Avertx-modular-router&pullRequest=1&issues=AWRfOMDDfzTos0U-AMY_&open=AWRfOMDDfzTos0U-AMY_)

[ISSUE_KEY]: AWRfOMDDfzTos0U-AMY_